Dockerfile

```diff
@@ -1,33 +1,17 @@
 FROM jenkinsci/jenkins:latest
```

Comment:
@michaelneale dogfood is using this image, basing the image on the LTS would install an older Jenkins version on dogfood. I'll keep it as is for now.